source/guides/administrator.rst

```diff
@@ -148,7 +148,7 @@ Learn how to keep Mattermost current with fixes and new features.
    /administration/upgrade.rst
    /administration/important-upgrade-notes*   
    /administration/changelog*
-   /administration/version-archive*
+   /administration/current-version*
```

Comment:
I'll discuss the naming of this with PMs.

Reply:
I wait until i push again.